ai_bridge/models/ai_bridge.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import json
+
+from odoo import api, fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class AiBridge(models.Model):
+
+    _name = "ai.bridge"
+    _description = "Ai Bridge Configuration"
+    _order = "sequence, id"
+
+    sequence = fields.Integer(
+        default=10,
+    )
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    description = fields.Text()
+    model_id = fields.Many2one(
+        "ir.model",
```

Comment:
Non-persistent models should be filtered out, as well as other security-related models like "res.users, res.groups,..."
How do you see it?

transient=False

Reply:
transient for sure. about ignoring other models, I am not so sure